README.md

```diff
@@ -98,101 +70,179 @@ Blaze4J bridges Java and native C++ for **fast, standards-compliant JSON Schema
+import com.github.madhavdhatrak.blaze4j.ValidationError;
+
+public class TestBlaze4j {
+    public static void main(String[] args) {
+        System.out.println("\nTesting enum validation:");
+        String enumSchemaJson = "{"
+            + "\"$schema\": \"https://json-schema.org/draft/2020-12/schema\","
+            + "\"enum\": [\"red\", \"green\", \"blue\"]"
+            + "}";
+
+        String invalidEnumInstance = "\"yellow\"";
+
+        try (CompiledSchema compiledSchema = Blaze.compile(enumSchemaJson)) {
+            ValidationResult result = Blaze.validateWithDetails(compiledSchema, invalidEnumInstance);
+
+            System.out.println("Is valid: " + result.isValid());
+
+            if (!result.isValid()) {
+                System.out.println("Validation errors:");
+                result.getErrors().forEach(System.out::println);
+            }
+        } catch (Exception e) {
+            System.err.println("Error: " + e.getMessage());
+            e.printStackTrace();
+        }
+    }
+}
 
-**Run the test suite for each draft:**
-```bash
-mvn test -Dtest=Draft2020Runner 
-```
-```bash
-mvn test -Dtest=Draft2019Runner 
-```
-```bash
-mvn test -Dtest=Draft7Runner 
-```
-```bash
-mvn test -Dtest=Draft6Runner 
-```
-```bash
-mvn test -Dtest=Draft4Runner 
 ```
+
 ---
+## Supported Drafts 
 
-## Usage
+| Draft Version | Status        | Specification                                                                 |
+|---------------|--------------|-------------------------------------------------------------------------------|
+| Draft 2020-12 | ✅ Supported | [Spec](https://json-schema.org/draft/2020-12/release-notes.html)              |
+| Draft 2019-09 | ✅ Supported | [Spec](https://json-schema.org/draft/2019-09/release-notes.html)              |
+| Draft 7       | ✅ Supported | [Spec](https://json-schema.org/draft-07/release-notes.html)                   |
+| Draft 6       | ✅ Supported | [Spec](https://json-schema.org/draft-06/release-notes.html)                   |
+| Draft 4       | ✅ Supported | [Spec](https://json-schema.org/draft-04/release-notes.html)                   |
 
-```java
+---
+<a name="supported-resolvers"></a>
+## Supported `$ref` Resolvers:
 
-String schemaJson = "{"
-    + "\"$schema\": \"https://json-schema.org/draft/2020-12/schema\","
-    + "\"type\": \"string\""
-    + "}";
+| Resolver Type | Status        | Description                                   |
+|---------------|---------------|-----------------------------------------------|
+| `http`        | ✅ Supported  | Resolves external schemas via HTTP(S) URLs    |
+| `classpath`   | ✅ Supported  | Resolves schemas from the Java classpath      |
 
-String instance = "\"hello\"";
+---
 
-try (CompiledSchema compiledSchema = Blaze.compile(schemaJson)) {
-    ValidationResult result = Blaze.validateWithDetails(compiledSchema, instance);
 
-    System.out.println("Is valid: " + result.isValid());
+## Dependencies
```

Comment:
This section would also be better to have in the other md file. Also make sure to check the JDK version (I assume 11 is a typo below, should be 21)